app/models/assignment.rb

```diff
@@ -46,6 +46,7 @@ class Assignment < ActiveRecord::Base
   # is no parent (the same holds for the child peer reviews)
   belongs_to :parent_assignment, class_name: 'Assignment', inverse_of: :pr_assignment
   has_one :pr_assignment, class_name: 'Assignment', foreign_key: :parent_assignment_id, inverse_of: :parent_assignment
+  has_many :peer_reviews, through: :groupings
```

Comment:
This isn't doing what I think it should. I assume we want to call this on the PR assignment, instead of on the parent? This one only works for the parent.